### DISCUSSION TITLE
Improve Helm chart ingress options

### DISCUSSION THREAD

**Discussion by Ramblurr** [nextcloud/all-in-one#2590] [Ideas] [answered]
### Is your feature request related to a problem? Please describe. 

The helm chart assumes  the user wants to expose the service via a `LoadBalancer`. But in production k8s clusters we have ingress' already configure that provide (amongst other things) SSL termination.

### Describe the solution you'd like

* The ability to configure an ingress with an `ingressClassName`, annotations, etc.

### Describe alternatives you've considered

There is no alternative except to fork the helm chart.

### Additional context

n/a

**Comment by szaimen** [comment, marked as answer]
Already asked in https://github.com/nextcloud/all-in-one/discussions/2589
